src/acquisition/covid_hosp/common/utils.py

```diff
@@ -178,8 +85,8 @@ def update_dataset(database, network):
 
     # get dataset details from metadata
     metadata = network.fetch_metadata()
-    url, revision = Utils.extract_resource_details(metadata)
-    issue = Utils.get_issue_from_revision(revision)
+    issue = max(metadata.index)
```

Comment:
What will happen here if HHS has posted more than one update since the last time we ran this code?

Reply:
For state timeseries and facility, it gets only the most recent one (the previous code with extract_resource_details seems to do the same, but perhaps I misunderstood the metadata values).

For the state_daily, it should grab all issues after the highest issue in the db and submit them as one issue dated as the latest issue.

Reply:
the previous API didn't give us a choice  -- it could only ever yield the metadata for the most recent issue.

Since the new API _does_ expose all issues, we should make sure we don't skip any.